scripts/jenkins-status.sh

```diff
@@ -1,12 +1,70 @@
-set -ex
+#!/usr/bin/env bash
 
-JENKINS_USERNAME=$1
-JENKINS_API_TOKEN=$2
-JENKINS_URL=$3
+### Setup script environment
+set -exuo pipefail
 
-printf "\n:jenkins: <https://build.platform.hmcts.net/|_*Jenkins Status*_> \n\n" >> slack-message.txt
+# Source central functions script
+source scripts/common-functions.sh
 
-BUILD_QUEUE_RESULT=$( curl -u "$JENKINS_USERNAME":"$JENKINS_API_TOKEN" "$JENKINS_URL/queue/api/json")
+slackBotToken=
+slackChannelName=
+jenkinsUsername=
+jenkinsApiToken=
+jenkinsURL=
+
+usage(){
+>&2 cat << EOF
+------------------------------------------------
+Script to check GitHub page expiry
+------------------------------------------------
+Usage: $0
+    [ -t | --slackBotToken ]
+    [ -c | --slackChannelName ]
+    [ -j | --jenkinsUsername ]
+    [ -t | --jenkinsApiToken ]
+    [ -u | --jenkinsURL ]
+    [ -h | --help ]
+EOF
+exit 1
+}
```

Comment:
```suggestion
usage(){
    >&2 cat << EOF
    ------------------------------------------------
    Script to check GitHub page expiry
    ------------------------------------------------
    Usage: $0
        [ -t | --slackBotToken ]
        [ -c | --slackChannelName ]
        [ -j | --jenkinsUsername ]
        [ -t | --jenkinsApiToken ]
        [ -u | --jenkinsURL ]
        [ -h | --help ]
    EOF
    exit 1
}
```

Small thing - just readability